packages/renderer/src/lib/network/network-utils.ts

```diff
@@ -33,6 +33,18 @@ export class NetworkUtils {
       engineType: networkInspectInfo.engineType,
       selected: false,
       status: Object.keys(networkInspectInfo.Containers ?? {}).length > 0 ? 'USED' : 'UNUSED',
+      containers: this.getNetworkContainers(networkInspectInfo),
+      ipv6_enabled: networkInspectInfo.EnableIPv6,
     };
   }
+
+  getNetworkContainers(networkInspectInfo: NetworkInspectInfo): NetworkContainer[] {
+    if (networkInspectInfo.Containers) {
+      return Object.keys(networkInspectInfo.Containers).map(containerId => {
+        return { name: networkInspectInfo.Containers?.[containerId].Name ?? '', id: containerId } as NetworkContainer;
```

Comment:
suggestion(non-blocking): avoid to cast when not needed, here you can use satisfies if you want

```suggestion
        return { name: networkInspectInfo.Containers?.[containerId].Name ?? '', id: containerId } satisfies NetworkContainer;
```